python/tests/api/writer/test_whylabs.py

```diff
@@ -231,6 +237,7 @@ def test_upload_reference_request(self, results):
             )
             assert response[0] is True
 
+    @pytest.mark.skip("Skip for now. Will need less mocking")
```

Comment:
There are a few of these tests that mock the code under test and thus don't test anything. I've left them in rather than just deleting them on the theory that we might come back later and mock the back end rather than mocking the entire interaction.